templates/default/image.tpl.html

```diff
@@ -1,3 +1,4 @@
-<div class="images"><img src='<%= file.url %>' alt='<%= description %>'>
-    <span class="image-caption"><%= caption %></span>
-</div>
\ No newline at end of file
+<div><img src='./img/<%= file.name %>' alt='<%= description %>'>
+    <span class="image-caption"><%= caption %>
+    </span>
+</div>
```

Comment:
I'm not a fan of the indentation in this file. Please break the line right after the `<div>`.
Also decide if you want the `<span>` and `</span>` on the same line as the caption or not (right now you mix the styles<).